pydal/adapters/base.py

```diff
@@ -1342,7 +1342,10 @@ def represent(self, obj, fieldtype):
             elif not isinstance(obj, (list, tuple)):
                 obj = [obj]
             if field_is_type('list:string'):
-                obj = map(str,obj)
+                try:
+                    obj = map(str,obj)
+                except UnicodeEncodeError:
+                    obj = map(lambbda x:unicode(x).encode(self.db_codec),obj)
```

Comment:
If you fix it in the branch of the pull request it updates automatically
On Apr 9, 2015 9:30 AM, "Tom Stratton" <notifications@github.com> wrote:

> In pydal/adapters/base.py
> <https://github.com/web2py/pydal/pull/131#discussion_r28066254>:
>
> > @@ -1342,7 +1342,10 @@ def represent(self, obj, fieldtype):
> >              elif not isinstance(obj, (list, tuple)):
> >                  obj = [obj]
> >              if field_is_type('list:string'):
> > -                obj = map(str,obj)
> > +                try:
> > +                    obj = map(str,obj)
> > +                except UnicodeEncodeError:
> > +                    obj = map(lambbda x:unicode(x).encode(self.db_codec),obj)
>
> Is there a way to resubmit a pull with the error fixed or do I just keep
> making new ones?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/web2py/pydal/pull/131/files#r28066254>.
>